opentelemetry-sdk/src/opentelemetry/sdk/metrics/_internal/instrument.py

```diff
@@ -130,11 +130,16 @@ def callback(
         for callback in self._callbacks:
             try:
                 for api_measurement in callback(callback_options):
-                    yield Measurement(
-                        api_measurement.value,
-                        instrument=self,
-                        attributes=api_measurement.attributes,
-                    )
+
+                    if api_measurement is None:
```

Comment:
How would `api_measurement` every be None?

Reply:
After discussing this, we no longer add this change.